Source/file.swift

```diff
@@ -0,0 +1,5 @@
+class xscShortVariableName {
```

Comment:
Type Name Violation: Type name should start with an uppercase character: 'xscShortVariableName' (type_name)